sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -0,0 +1,144 @@
+"""Testing for the boost module (sklearn.ensemble.boost)."""
+
+import numpy as np
+
+from sklearn.utils.testing import assert_almost_equal
+from sklearn.utils.testing import assert_raises, assert_raises_regexp
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.naive_bayes import GaussianNB
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.ensemble import VotingClassifier
+from sklearn import datasets
+from sklearn import cross_validation
+from sklearn.datasets import make_multilabel_classification
+from sklearn.svm import SVC
+from sklearn.multiclass import OneVsRestClassifier
+
+# Load the iris dataset and randomly permute it
+iris = datasets.load_iris()
+X, y = iris.data[:, 1:3], iris.target
+
+
+def test_majority_label_iris():
+    """Check classification by majority label on dataset iris."""
+    clf1 = LogisticRegression(random_state=123)
+    clf2 = RandomForestClassifier(random_state=123)
+    clf3 = GaussianNB()
+    eclf = VotingClassifier(estimators=[clf1, clf2, clf3], voting='hard')
+    scores = cross_validation.cross_val_score(eclf,
+                                              X, 
+                                              y, 
+                                              cv=5, 
+                                              scoring='accuracy')
+    assert_almost_equal(scores.mean(), 0.95, decimal=2)
+
+
+def test_tie_situation():
+    """Check voting classifier selects smaller class label in tie situation."""
+    clf1 = LogisticRegression(random_state=123)
+    clf2 = RandomForestClassifier(random_state=123)
+    eclf = VotingClassifier(estimators=[clf1, clf2], voting='hard')
+    assert(clf1.fit(X,y).predict(X)[73] == 2) 
+    assert(clf2.fit(X,y).predict(X)[73] == 1) 
+    assert(eclf.fit(X,y).predict(X)[73] == 1) 
+
```

Comment:
please run the pep8 linter on your new files. In particular here there should be 2 blank lines between top level blocks.